src/chapters/classes-part2/class-4-objectives.rst

```diff
@@ -0,0 +1,14 @@
+Java Lesson 4
+==============
+
+Missing Objectives?
+---------------------
+
+The following are included in the LC101 unit objectives for lesson 4, but
+they may not be fully implemented in the old curriculum. Please keep these in
+mind as you review the chapter content.
+
+#. Explain why instance variables may not be referenced within static methods
+#. Explain when .hashCode should be implemented within a class
+#. Use IntelliJ’s code generation tools to generate well-structured .equals
+   and .hashCode methods
```

Comment:
These objectives should be incorporated into the chapter. I didn't see them in the chapter beyond this page, but I may have missed it.

Reply:
If you missed it, then we both did.  I flagged these because I do not think they have been fully addressed.  Suggestions?